Spigot-API-Patches/0299-Add-corner-getter-to-WorldBorder.patch

```diff
@@ -0,0 +1,57 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TwoLeggedCat <80929284+TwoLeggedCat@users.noreply.github.com>
+Date: Wed, 12 May 2021 20:07:03 -0500
+Subject: [PATCH] Add corner getter to WorldBorder
+
+
+diff --git a/src/main/java/org/bukkit/Corner.java b/src/main/java/org/bukkit/Corner.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..66d6edf6269e4be74e8b329df5607b136b76e548
+--- /dev/null
++++ b/src/main/java/org/bukkit/Corner.java
+@@ -0,0 +1,27 @@
++package org.bukkit;
```

Comment:
I'd add this to the bukkit Worldborder class itself, or at least it should be in the io.papermc.paper package.

Reply:
Will add to WorldBorder.